src/child/containers/sidebars/favourites/Favourites.js

```diff
@@ -41,6 +42,10 @@ class Favourites extends Component {
         // TODO: send content to tearout
     }
 
+    onQuandlResponse(stockCode, stockName) {
+        this.props.dispatch(quandlResponse(stockCode, stockName));
```

Comment:
Since we now have the codes and names stored, we can afford to only pass codes around.